### ISSUE TITLE
Need instructions on how to deploy new website

### ISSUE THREAD

**Issue by shazron** [phonegap/phonegap-day#96]
Related to https://github.com/phonegap/discuss/issues/40 (further private discussion there)

Need:
1. Instructions on how to deploy new website
2. Instructions to get credentials to deploy

**Comment by shazron** [comment]
Went into settings - turns out the docs are through gh-pages, but through the "docs" folder on master branch.

**Comment by GarthDB** [comment]
Yes, anything we deploy to the docs folder on master is used as a static site. There is an npm script to build the production directory `build:production` I'll put it in the readme
